File: samples/client/petstore/java/retrofit2-play26/src/main/java/org/openapitools/client/api/DefaultApi.java
package org.openapitools.client.api;

import org.openapitools.client.CollectionFormats.*;

import retrofit2.Call;
import retrofit2.http.*;

import okhttp3.RequestBody;
import okhttp3.ResponseBody;
import okhttp3.MultipartBody;

import org.openapitools.client.model.Dummy200Response;
import org.openapitools.client.model.ExampleResponse;

import java.util.ArrayList;
import java.util.HashMap;
import java.util.List;
import java.util.Map;

import java.util.concurrent.*;
import retrofit2.Response;

public interface DefaultApi {
  /**
   * 
   * dummy
   * @return Call&lt;Dummy200Response&gt;
   */
  @GET("example/someMethod")
  CompletionStage<Response<Dummy200Response>> dummy();
    

}

File: samples/client/petstore/kotlin-default-values-jvm-retrofit2/src/main/kotlin/org/openapitools/client/apis/DefaultApi.kt
package org.openapitools.client.apis

import org.openapitools.client.infrastructure.CollectionFormats.*
import retrofit2.http.*
import retrofit2.Call
import okhttp3.RequestBody
import com.squareup.moshi.Json

import org.openapitools.client.models.Dummy200Response

interface DefaultApi {
    /**
     * 
     * dummy
     * Responses:
     *  - 200: dummy
     *
     * @return [Call]<[Dummy200Response]>
     */
    @GET("example/someMethod")
    fun dummy(): Call<Dummy200Response>

}
